user_guide_src/source/libraries/curlrequest/034.php

```diff
@@ -0,0 +1,3 @@
+<?php
+
+$response->request('GET', 'http://example.com', ['debug' => true]);
```

Comment:
This is not your mistake, but isn't it `$client->request(`?

Reply:
Also all the other sample code in [Request Options](https://codeigniter4.github.io/CodeIgniter4/libraries/curlrequest.html#request-options).

Reply:
updated